helion/_testing.py

```diff
@@ -98,7 +98,12 @@ def run_example(
     for name, func in {**kernels, **baselines}.items():
         if name != first_baseline_name:
             print(f"Testing {name} correctness...", file=sys.stderr)
-            torch.testing.assert_close(func(*args), expected, rtol=rtol, atol=atol)
+            torch.testing.assert_close(
+                func(*args).to(torch.float32),
+                expected.to(torch.float32),
+                rtol=rtol,
+                atol=atol,
+            )
```

Comment:
Without casting to fp32, it errors with `Rtol=0.1 and atol=0.1 are not supported for bitwise comparison of low dimensional floats. Please use rtol=0.0 and atol=0.0.`